crypto/ec/ec_ameth.c

```diff
@@ -303,12 +303,15 @@ static int ec_copy_parameters(EVP_PKEY *to, const EVP_PKEY *from)
     if (to->pkey.ec == NULL) {
```

Comment:
May put a space line below variable declaration. We follow a "Fix-one-also-fix-the-neighbour' policy.